test/functionalTest/cases/3005_GET_registrations/get_registrations.test

```diff
@@ -0,0 +1,379 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+GET registration using APIv2, created also using V2
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Register entity E1/A1, with service path /a1/e1
+# 02. Register entity E2/A1, with service path /a1/e2
+# 03. Register entity E3/A1, with service path /a1/e3
+# 04. Register entity E4/A1, with service path /#
+# 05. GET all four registrations, without service path
+# 06. GET no registration, with service path /a1/# ...
+# 07. GET one registration using service path /a1/e2
+# 08. GET no registration using service path /#
```

Comment:
Actually, based on the exact match principle, it should return registration for E4/E1

Reply:
That is so true ...
Apparently, I was in a hurry this morning. Sorry about that!!!

Reply:
Fixed in 725e934